docs/.vuepress/config.js

```diff
@@ -19,6 +19,15 @@ module.exports = {
             },
           ]
         },
+        {
+          title: "Platform",
```

Comment:
should this be Guide or Tutorial instead?

Reply:
The docs are primarily developer-facing, so a Guide/Tutorial section that doesn't include anything related to development feels off. Future PRs will expand this section with other platform related content.

Reply:
Open to opinions on this one